docs/ref/databases.txt

```diff
@@ -103,7 +103,7 @@ PostgreSQL notes
 ================
 
 Django supports PostgreSQL 9.4 and higher. `psycopg2`_ 2.5.4 through 2.8.6 is
-required, though the latest release is recommended.
+required, though the 2.8.6 is recommended.
```

Comment:
`though the 2.8.6` => `though 2.8.6`?

Reply:
Updated.

Reply:
Maybe I've not had enough coffee yet. It looks like should chop the `the` (as per Keryn's comment?)

Reply:
Yes, sorry. `push` without `amend` :facepalm: